File: core/py/Candle.py
class Candle:
    def __init__(self,time,period,open,high,low,close,volume,symbol=None):
        self.time=time
        self.period=period
        self.open=open
        self.high=high
        self.low=low
        self.close=close
        self.volume=volume
        self.symbol=symbol
    @staticmethod
    def get(index,period,symbol):
        c=Candle(0,period,0,0,0,0,0,symbol)
        return c

File: core/js/Candle.js
class Candle{
  constructor(time,period,open,high,low,close,volume,symbol=null){
    this.time=time; this.open=open; this.high=high; this.low=low; this.close=close; this.volume=volume;
    this.symbol=symbol; this.period=period;
  }
  static get(symbol,period,index){
    return Context.activeObject.getCandle(symbol,period,index);
  }
}
